### PR TITLE
Updated Sign Up and Login

### PR DESCRIPTION
**Pull request by SenaiAyalew**

### CODE REVIEW

File: js/Login.js [hunk 1]
```diff
@@ -82,14 +82,25 @@ export default class Login extends Component {
   }
 
   _submit() {
+    // const url1 = 'http://ec2-34-238-240-14.compute-1.amazonaws.com/login';
+    const testServer = 'http://192.168.0.15:8200/login';
     axios({
       method: 'post',
-      url: 'http://ec2-34-238-240-14.compute-1.amazonaws.com/login',
-      data: this.state,
+      url: testServer,
```
Comment (Alecgilchrist) [inline js/Login.js]: test server still hardcoded

File: js/Signup.js [hunk 2]
```diff
@@ -92,15 +92,24 @@ export default class Signup extends Component {
...
+      data: {
+        username: this.state.username,
+        email: this.state.email,
+        password: this.state.password,
+      },
     })
       .then((response) => {
-        // console.warn('response', response);
-        this.props._logIn();
+        console.warn(123456789, response);
```
Comment (Alecgilchrist) [inline js/Signup.js]: console warn. Do we still need this? is this still in progress?

File: server.js [hunk 3]
```diff
@@ -152,25 +177,51 @@ app.get('/broad', (req, res) => {
...
 app.post('/login', (req, res) => {
-  // TODO: Senai do some stuff here
-  console.log('server post login endpoint');
-  console.log(req.body, 'rrreeeqqqq......bbbbooooddddyyyy');
-  // helpers.loginUser(req, res);
-  // helpers.createUser(req, res);
-  res.send(req.body);
-  // res.send('logged in');
+  const userInfo = req.body;
+  console.log(userInfo, '88888888888888888');
```
Comment (Alecgilchrist) [inline server.js]: long console logs. again still in progress?

File: server.js [hunk 4]
```diff
@@ -152,25 +177,51 @@ app.get('/broad', (req, res) => {
...
-  console.log('signUp user fired');
-  // console.log('user: ', user);
-  console.log(req.body);
-  res.send(req.body);
+app.post('/signup', (req, res) => {
+  const userObject = req.body;
+  dbHelpers.findUserSignup(userObject)
+    .then((response) => {
+      console.log('server /signup #############', response);
+      if (response === '1') {
```
Comment (Alecgilchrist) [inline server.js]: what is '1' and '2' are these testing values or do are they a ref to something?

File: database-mySql/dbHelpers.js [hunk 5]
```diff
@@ -5,43 +5,60 @@ const {
...
+    console.log('dbHelpers => findUserSignup => then() !!!!!!!!!!!!!!!', user);
+    if (user === null) {
+      User.create({
+        where: {
+          username: userInfo.username,
+          email: userInfo.email,
+          password: userInfo.password,
+        },
+      });
+      return '1';
```
Comment (Alecgilchrist) [inline database-mySql/dbHelpers.js]: why 1, 2? is this needed or testing

File: database-mySql/index.js [hunk 6]
```diff
@@ -106,7 +107,7 @@ User.sync({
 })
   .then(() => {
     // add the same user again to test function - should return false
-    User.findOrCreate({ where: { userName: 'Josef', email: 'email@email.com' } })
+    User.findOrCreate({ where: { username: 'Josef', email: 'email@email.com' } })
```
Comment (Alecgilchrist) [inline database-mySql/index.js]: hardcoded to one user

File: database-mySql/dbHelpers.js (hunk 5 above)
Comment (SenaiAyalew) [reply database-mySql/dbHelpers.js]: Definitely needed

File: database-mySql/index.js (hunk 6 above)
Comment (SenaiAyalew) [reply database-mySql/index.js]: I have never touched this file, talk to Joe about this. He did this. This just puts one user in the database every time we start the server/ database. To ensure we have atleast one test dummy.

File: js/Login.js (hunk 1 above)
Comment (SenaiAyalew) [reply js/Login.js]: Gotcha

File: js/Signup.js (hunk 2 above)
Comment (SenaiAyalew) [reply js/Signup.js]: I had been working on Login and forget to clean up Signup. we do not need this. Signup and Login are completely finsihed, minus tokens and refresh.

File: server.js (hunk 3 above)
Comment (SenaiAyalew) [reply server.js]: Nope, not in progress, I just forgot to remove it. Cleaning it up now.

File: server.js (hunk 4 above)
Comment (SenaiAyalew) [reply server.js]: #1 means the user signing up has a valid email address (an email address that is not already in the database)

#2 means that the email the user tries signing up with already exists in the database. If you look in the if block you can see the response to the client are different.

I definitely need these, they are not for testing. The is the response I am sending form the server. I can make them true and false if you prefer.

**Pull request merged**